src/io/import_svg.cc

```diff
@@ -27,7 +27,7 @@
 #include <Eigen/Core>
 #include <Eigen/Geometry>
 
-#include "import.h"
+#include "ioimport.h"
```

Comment:
Are these renames needed, or could you revert that to make this PR cleaner?

Reply:
Right, python header file conflict..
I'm wondering whether we should just nudge this towards a cleaner solution, by e.g. removing "src/io" from OpenSCAD's `-I` compiler arg and instead `#include "io/import.h". Otherwise we're ending up playing some sort of cat and mouse game with external headers.

Reply:
This belong in a separate PR though, if we decide to do it, so there's nothing actionable here.

Reply:
Yep, I was wondering about this too. Sounds like it's good to go for now, but something to consider simetime later for some better header file structure ideally with a single top-level `-I`.

Reply:
Perhaps Better would be #include <openscad/xxx/yyy.h> which gives openscad its own namespace